src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -206,7 +215,8 @@ void ProbeDirectory(string manifestDirectory)
         private (string? id, string? manifestDirectory) ResolveManifestDirectory(string manifestDirectory)
         {
             string manifestId = Path.GetFileName(manifestDirectory);
-            if (_outdatedManifestIds.Contains(manifestId))
+            if (_outdatedManifestIds.Contains(manifestId) ||
+                manifestId.Equals(WorkloadSetsFolderName, StringComparison.OrdinalIgnoreCase))
```

Comment:
When/why does this get called with the `WorkloadSetsFolderName`? Is this just a safety measure / defensive programming?

Reply:
The `workloadsets` folder is a sibling to the workload manifest folders, but should be ignored when looking for workload manifests, as it stores different information.

Reply:
Thanks, that makes sense and I agree with that. I meant to ask since it _should_ be ignored, in what contexts would this function be called with a `manifestDirectory` that has the foldername `workloadsets`? Because we should never do that 'ideally,' right? Do we expect that to ever happen, or is this a defensive mechanism against it? Either way, seems OK, but I wanted to know.

Reply:
This method will actually get called for the `workloadsets` folder.  It needs to get filtered out somewhere, and this method seemed like the best place to do it because then the filtering doesn't need to be duplicated in the multiple places that call this method.

Reply:
Ah yes, I see how it happens now. Thank you :)